### DISCUSSION TITLE
ECIES Decryption

### DISCUSSION THREAD

**Discussion by danielherrero** [keplerproject/luarocks#1719] [General]
Hi
I am working on a wireshark dissector using LUA and I need to decrypt frames with ECIES decryption. I am learning lua and I have watched ec25519.lua but I dont know if It can be handled using this libray. Is it possible?
Thanks in advance

**Comment by alerque** [comment]
This is the issue tracker / discussion form for `luaRocks` the package management tool. It is not a general purpose place to discuss any of the projects listed on the luarocks.org website or elsewhere in the ecosystem. You should open an issue an the repository for the project you are actually inquiring about. Perhaps you mean to inquire on [luasec255519](https://luarocks.org/modules/ssmmxx/luaec25519) or [plc](https://github.com/philanc/plc)?

**Comment by Tieske** [comment]
If you're starting to learn Lua, then probably best to ask this on the Lua mailing list. The authors/contributors of `plc` are on that list iirc.
